osu.Game.Rulesets.Osu.Tests/TestSceneHitMarker.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable disable
+
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using osu.Game.Replays;
+using osu.Game.Rulesets.Osu.Replays;
+using osu.Game.Rulesets.Osu.UI;
+using osu.Game.Rulesets.Replays;
+using osu.Game.Tests.Visual;
+using osuTK;
+
+namespace osu.Game.Rulesets.Osu.Tests
+{
+    public partial class TestSceneHitMarker : OsuTestScene
```

Comment:
The test should create a replay which lasts indefinitely, or at least long enough to see and work on the visuals of this component. Right now it lasts like 500 ms and disappears before you can even see it.